lib/cli.js

```diff
@@ -100,13 +97,31 @@ const onLoad = async (targetDir, version, argv) => {
     }
     lang = langChoice || "";
   }
+  const isJqueryIncluded =
+    version !== "latest" && compareVersions(version.toString(), "8.0.0"); // jQuery removed in >= v8
```

Comment:
Is there a reason to call `toString ()` on version when passing it to `compareVersions ()`? Is it not aways a string? If it is not what can it be such that calling `toString ()` will return something sensible?

Reply:
`node index.js ./out/vlad -r=6` in that case version is 6 (type of Number)

Reply:
But actually It would be better if `version` always would be string, as I don't need Number. So I can just store it as a string in the first place. Thanks for pointing me to this.